### PR TITLE
Vs 629 failure to retrieve job information during ingest

### PR DESCRIPTION
**Pull request by koncheto-broad**
hardening standard quickstart pipeline against errors when a location is specified

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]
Looks great! Only a couple of very minor nits

File: scripts/variantstore/utils/pushGATKtoGCS.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+DT=$(date '+%Y%m%d')
+
+branch=$(git symbolic-ref HEAD 2>/dev/null)
+branch=${branch#refs/heads/} 
+
+DEST="gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/${branch}_${DT}/"
+
+gsutil cp $JAR $DEST
+
+echo "Copied to $DEST$(basename $JAR)"
```
[the remark this answers is not in the document]
Comment (koncheto-broad) [reply scripts/variantstore/utils/pushGATKtoGCS.sh]: got it

File: scripts/variantstore/wdl/GvsAssignIds.wdl [hunk 2]
```diff
@@ -198,11 +198,11 @@ task CreateCostObservabilityTable {
       PARTITION_STRING="--time_partitioning_field call_start_timestamp --time_partitioning_type DAY"
       echo "making table $TABLE"
       echo '~{cost_observability_json}' > schema.json
-      bq --location=US mk ${PARTITION_STRING} --project_id=~{project_id} $TABLE schema.json
+      bq mk ${PARTITION_STRING} --project_id=~{project_id} $TABLE schema.json
     fi
   >>>
   runtime {
-    docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:2022-10-06-alpine"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsAssignIds.wdl:205]: couldn't this stay `gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine` (which is smaller)?

**Review by mcovarr — APPROVED** [review]
one minor update requested

File: scripts/variantstore/wdl/GvsAssignIds.wdl (hunk 2 above)
Comment (koncheto-broad) [reply scripts/variantstore/wdl/GvsAssignIds.wdl]: I thought I needed to change that one earlier, but it actually runs just fine using the smaller version of the file.  Good catch!

**Pull request merged**